esx_service/utils/kvESX.py

```diff
@@ -42,24 +47,78 @@
 # Default kv side car alignment
 KV_ALIGN = 4096
 
+global using_64bits
+
 # Load the disk lib API library
-def loadDiskLib():
+def loadDiskLib(lib_name):
     global lib
 
     if not lib:
-        lib = CDLL(DISK_LIB)
-        lib.DiskLib_Init.argtypes = []
-        lib.DiskLib_Init.restype = c_bool
-        lib.DiskLib_Init()
+       lib = CDLL(lib_name)
+       lib.DiskLib_Init.argtypes = []
+       lib.DiskLib_Init.restype = c_bool
+       lib.DiskLib_Init()
 
 
 # Loads the ESX library used to access proprietary ESX disk lib API.
 # Create arg/result definitions for those that we use.
-def kvESXInit():
+
+# Initialize 32-bit disk lib interfaces
+def kv_ESX_lib64_init():
```

Comment:
why 64 bit init is different from 32 bit init? Looking into function defintion, they seem to using 32 or 64 bit explicitly, so why different functions? 

[nit] if it **is** needed, I'd prefer to hide this in kv_esx_lib_init() rather that having it done upstairs